src/components/ControlBar/ViewControlBar/styles/DashboardsBar.module.css

```diff
@@ -0,0 +1,112 @@
+.container {
+    position: relative;
+    background-color: var(--colors-white);
+    box-shadow: rgba(0, 0, 0, 0.2) 0px 0px 6px 3px;
+    z-index: 2001;
+    overflow: hidden;
+    box-sizing: border-box;
+    flex: none;
+    display: flex;
+    flex-direction: column;
+}
+
+.content {
+    padding: 10px 6px 0 6px;
+    overflow: hidden;
+    margin-bottom: 21px; /* to make space for the show more button */
+}
+
+.expanded .content {
+    overflow-y: auto;
+}
+
+.one {
```

Comment:
Not a huge fan of these. See my other comment above. However, as they only seem to increment with by 40, couldn't this just be generated somehow instead? `height = 31 + (40 * rowCount)`?

Reply:
Thanks for the tip. Code improved (and learning more about using css vars).